guides/common/assembly_installing-satellite-server-disconnected.adoc

```diff
@@ -0,0 +1,45 @@
+[id="installing-satellite-server-disconnected"]
+= Installing {ProjectServer}
+
+When the intended host for {ProjectServer} is in a disconnected environment, you can install {ProjectServer} by using an external computer to download an ISO image of the packages, and copying the packages to the system you want to install {ProjectServer} on. This method is not recommended for any other situation as ISO images might not contain the latest updates, bug fixes, and functionality.
+
+Use the following procedures to install {ProjectServer}, perform the initial configuration, and import subscription manifests.
+
+ifeval::["{build}" != "foreman-deb"]
+Before you continue, consider which manifests are relevant for your environment. For more information on manifests, see link:{BaseURL}content_management_guide/managing_subscriptions/[Managing Subscriptions] in the _Content Management Guide_.
+endif::[]
+
+
+NOTE: You cannot register {ProjectServer} to itself.
+
+include::modules/proc_downloading-the-binary-dvd-images.adoc[leveloffset=+1]
+
+include::modules/proc_configuring-the-base-operating-system-with-offline-repositories.adoc[leveloffset=+1]
+
+include::modules/proc_installing-from-the-offline-repositories.adoc[leveloffset=+1]
+
+include::modules/proc_resolving-package-dependancy-errors.adoc[leveloffset=+1]
+
+include::modules/proc_synchronizing-the-system-clock-with-chronyd.adoc[leveloffset=+1]
+
+include::modules/proc_installing-the-sos-package.adoc[leveloffset=+1]
+
+[id="configuring-server_{context}"]
+== Configuring {ProjectServer}
```

Comment:
Why is this not in a separate assembly?

Reply:
Also, the content of Configuring {ProjectServer} for connected and disconnected is vry similar, seems like there is a chance to reuse.

Reply:
I considered putting it in a concept file, but it would only ever be used in this context, so I think putting it directly in the installing-satellite-server assembly is the convention in that case (or it was last time I did the training). I'll have a think about it though.

Reply:
Thank you for the explanation